build/localnet/private-keys.yaml

```diff
@@ -0,0 +1,1014 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: v1-localnet-validators-private-keys
+  namespace: default
+type: Opaque
+stringData:
+  # OLD keys for reference
+  # val1: "c6c136d010d07d7f5e9944aa3594a10f9210dd3e26ebc1bc1516a6d957fd0df353ee26c82826694ffe1773d7b60d5f20dd9e91bdf8745544711bec5ff9c6fb4a" # 113fdb095d42d6e09327ab5b8df13fd8197a1eaf
+  # val2: "b37d3ba2f232060c41ba1177fea6008d885fcccad6826d64ee7d49f94d1dbc49a8b6be75d7551da093f788f7286c3a9cb885cfc8e52710eac5f1d5e5b4bf19b2" # 3f52e08c4b3b65ab7cf098d77df5bf8cedcf5f99
+  # val3: "5db3e9d97d04d6d70359de924bb02039c602080d6bf01a692bad31ad5ef93524c16043323c83ffd901a8bf7d73543814b8655aa4695f7bfb49d01926fc161cdb" # 67eb3f0a50ae459fecf666be0e93176e92441317
+  # val4: "6fd0bc54cc2dd205eaf226eebdb0451629b321f11d279013ce6fdd5a33059256b2eda2232ffb2750bf761141f70f75a03a025f65b2b2b417c7f8b3c9ca91e8e4" # 6f66574e1f50f0ef72dff748c3f11b9e0e89d32a
+
+  # New keys that start with a number that corresponds to a number of a validator. First two letters after the digits correspond to a type of actor. aa, being the first actor transitioning to this, is the validator.
```

Comment:
I realized that `aa` is probably not the best choice considering we already have `ActorType` attribute. So I'll regenerate the addresses to reflect actor type we already settled on.